content/v2/openapi.yml

```diff
@@ -1287,6 +1287,75 @@ paths:
         '412':
           $ref: '#/components/responses/412-feature'
       summary: De-delegate from vanity name servers
+  '/{account}/registrar/domains/{domain}/transfer_lock':
+    get:
+      description: Gets the transfer lock status for a domain.
+      parameters:
+        - $ref: '#/components/parameters/Account'
+        - $ref: '#/components/parameters/Domain'
+      operationId: getDomainTransferLock
+      tags:
+        - registrar
```

Comment:
We should delete this tag as per https://github.com/dnsimple/dnsimple-developer/pull/523